python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -152,13 +173,10 @@ interface {{ ethernet_interface.name }}
 {%     elif ethernet_interface.type is arista.avd.defined('switched') %}
    switchport
 {%     endif %}
-{%     if ethernet_interface.trunk_private_vlan_secondary is arista.avd.defined(true) %}
-   switchport trunk private-vlan secondary
-{%     elif ethernet_interface.trunk_private_vlan_secondary is arista.avd.defined(false) %}
-   no switchport trunk private-vlan secondary
-{%     endif %}
-{%     if ethernet_interface.pvlan_mapping is arista.avd.defined %}
-   switchport pvlan mapping {{ ethernet_interface.pvlan_mapping }}
+{%     if ethernet_interface.switchport.enabled is arista.avd.defined(true) %}
+   switchport
+{%     elif ethernet_interface.switchport.enabled is arista.avd.defined(false) %}
+   no switchport
 {%     endif %}
 {%     if ethernet_interface.vlan_id is arista.avd.defined and ethernet_interface.type | arista.avd.default != 'l2dot1q' %}
    vlan id {{ ethernet_interface.vlan_id }}
```

Comment:
s1-leaf1(config-if-Et10.10)#sh active 
interface Ethernet10.10
   encapsulation dot1q vlan 10
   vlan id 10

Reply:
Can not fix this since `vlan id` comes after `encapsulation dot1q vlan 10` but before `encapsulation vlan
      client unmatched`.
In j2 both of these are written in same if, elif block.

Reply:
Fixed!